lua/autorun/client/cl_cfc_restart_display.lua

```diff
@@ -0,0 +1,64 @@
+local finalTime
+local keyFrame = 0
+local isOpen = false
+local flashState = false
+
+surface.CreateFont( "CFC_CALIBRI_20PX_DEFAULT", {
+    font = "Calibri",
+    extended = false,
+    size = 40,
+    weight = 500
+})
+
+local function timeLeft()
+    return finalTime - SysTime()
+end
+
+concommand.Add( "clean", function()
+    isOpen = false
+end )
+
+local function restartUI()
+    timer.Create( "CFC_RESTART_FLASHTIMER", 1, 0, function()
+        flashState = not flashState
+    end )
+
+    isOpen = true
+    hook.Remove( "HUDPaint", "CFCDrawRestartAlert" )
```

Comment:
```suggestion
```
No need for this here :)

Reply:
Doing `hook.Add` with the same event and name replaces the hook, so theres no need to remove it if you're adding a new one back later.